RELEASE_PROCESS.md

```diff
@@ -0,0 +1,27 @@
+# How to cut a peg release
+
+This document is aimed at members of the Pony team who might be cutting a release of Pony. It serves as a checklist that can take you through doing a release step-by-step.
```

Comment:
Should this say "cutting a release of Pony"?

Reply:
Probably not. All the other libraries do right now. Lol.

Thoughts on how to change that in general?

Reply:
Is change to "peg" good enough?

Reply:
Sure, either that or "a ponylang project" if you want to be more general